app/javascript/campaign-form-v3.js

```diff
@@ -0,0 +1,141 @@
+// Only initialize campaign-forms JS if it hasn't loaded previously.
+if (typeof window.campaignForms === 'undefined') {
+
+  // Campaign Forms
+  (function ($) {
+    var campaignForms = window.campaignForms = window.campaignForms || {}
+    var campaignForm = window.campaignForm = window.campaignForm || {}
+    var idCounter = 0
+
+    campaignForms.jQuery = $.noConflict(true)
+
+    function uniqueFormId () {
+      var id = ++idCounter
+      return 'campaignForm' + id
+    }
+
+    function submitForm(form, recaptchaToken) {
+      var formId = form.attr('id')
+
+      if (!campaignForms[formId].formSubmitted) {
+        campaignForms[formId].formSubmitted = true
+        form.ajaxSubmit({
+          method: 'POST',
+          dataType: 'json',
+          data: { "g-recaptcha-response": recaptchaToken },
+          success: function (data, status, xhr) {
+            // Submit Adobe Analytics event if present
+            if (typeof window._satellite !== 'undefined') {
+              // Extend data layer with master_person_id
+              if (typeof data.master_person_id !== 'undefined')
+                window.digitalData = $.extend(true, {}, window.digitalData || {}, {
+                  user: [{profile: [{profileInfo: {grMasterPersonId: data.master_person_id}}]}]
+                })
+              // Extend data layer with campaign_code
+              if (typeof data.campaign_codes !== 'undefined' && data.campaign_codes.length > 0)
+                window.digitalData = $.extend(true, {}, window.digitalData || {}, {
+                  page: {pageInfo: {emailList: 'ACS | ' + data.campaign_codes.join(' | ')}}
```

Comment:
You could use template literals
```suggestion
                  page: { pageInfo: { emailList: `ACS | ${data.campaign_codes.join(' | ')}` } }
```

Reply:
This is a copy of campaign.js, is it worth doing this in both files?  @knutsenm what do you think?